spec/rails_helper.rb

```diff
@@ -22,7 +22,11 @@
 
   config.include Rails.application.routes.url_helpers
 
+  config.include OAuthHelpers
+
   config.include Devise::Test::IntegrationHelpers, type: :system
 
   config.include DeviseHelpers::SystemHelpers
+
+  config.include OAuthHelpers
```

Comment:
We already added it 😄

Reply:
I hate rebase... >< 
And trust that I've been mega careful in every conflict merging... I'm wondering if it simply merged it without seen the conflict (I probably added this line in 2 earlier branches).